package.json

```diff
@@ -114,6 +114,12 @@
   "publishConfig": {
     "access": "public"
   },
+  "kaltura": {
+    "name": "ima",
+    "dependencies": {
+      "playkit-timeline": "3.0.0"
```

Comment:
@amirch1 
should it be timeline or playkit-timeline ? assuming it uses "name", it's timeline 
https://github.com/kaltura/playkit-js-timeline/blob/8eb0755ca49dbca6fb99e5fb2c40e846a2040a5d/package.json#L114

Reply:
@SivanA-Kaltura I used https://github.com/kaltura/playkit-js-ivq/blob/master/package.json#L65 as reference. This works fine for IVQ. The name is not the plugin name but the bundle name